<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New_World_Trading_Post</class>
 <widget class="QWidget" name="New_World_Trading_Post">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>508</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/NW-coins-96x.ico</normaloff>../resources/NW-coins-96x.ico</iconset>
  </property>
  <widget class="QLabel" name="sell_label">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>94</y>
     <width>46</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Selling</string>
   </property>
  </widget>
  <widget class="QComboBox" name="buy_comboBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>40</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Charcoal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Iron Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Starmetal Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Orichalcum Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Silver Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gold Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Platinum Ore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Iron Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Steel Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Starmetal Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Orichalcum Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Silver Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gold Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Platinum Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stone</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stone Block</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stone Brick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lodestone</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lodestone Brick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Obsidian Voidstone</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fibers</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Silk Threads</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wirefiber</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Linen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sateen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Silk</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Infused Silk</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Green Wood</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Aged Wood</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Timber</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lumber</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wyrdwood</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ironwood</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wyrdwood Plank</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ironwood Plank</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rawhide</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thick Hide</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Iron Hide</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Coarse Leather</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rugged Leather</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Layered Leather</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Infused Leather</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sandpaper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tannin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Crossweave</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Flux</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Arcane Lodestone</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="buy_quantity_label">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>9</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QTreeView" name="debugTreeView">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>125</y>
     <width>561</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="buyFlip">
   <property name="geometry">
    <rect>
     <x>576</x>
     <y>40</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="sell_comboBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>94</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Charcoal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Iron Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Steel Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Starmetal Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Orichalcum Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Silver Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gold Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Platinum Ingot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stone Block</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stone Brick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lodestone Brick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Obsidian Voidstone</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Linen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sateen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Silk</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Infused Silk</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Timber</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lumber</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wyrdwood Plank</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ironwood Plank</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Coarse Leather</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rugged Leather</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Layered Leather</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Infused Leather</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="sell_quantity_label">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>71</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sellQuantity">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>94</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sell_individual_label">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>71</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Individual</string>
   </property>
  </widget>
  <widget class="QLabel" name="capital_label">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>9</y>
     <width>48</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Capital</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="buyQuantity">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>40</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sell_flip_label">
   <property name="geometry">
    <rect>
     <x>576</x>
     <y>71</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Flip Profit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="buyIndividual">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>40</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="buy_flip_label">
   <property name="geometry">
    <rect>
     <x>576</x>
     <y>9</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Flip Profit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sellFlip">
   <property name="geometry">
    <rect>
     <x>576</x>
     <y>94</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="buy_label">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>40</y>
     <width>47</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Buying</string>
   </property>
  </widget>
  <widget class="QLabel" name="buy_individual_label">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>9</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Individual</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sellIndividual">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>94</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Capital">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>576</x>
     <y>125</y>
     <width>161</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/StrillesCedias.xpm</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text_info">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>477</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Please familarize yourself with the README and instructions for use.  </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
